src/video/emscripten/SDL_emscriptenevents.c

```diff
@@ -409,7 +409,22 @@ static EM_BOOL
 Emscripten_HandleWheel(int eventType, const EmscriptenWheelEvent *wheelEvent, void *userData)
 {
     SDL_WindowData *window_data = userData;
-    SDL_SendMouseWheel(window_data->window, 0, (float)wheelEvent->deltaX, (float)-wheelEvent->deltaY, SDL_MOUSEWHEEL_NORMAL);
+
+    float deltaY = wheelEvent->deltaY;
+
+    switch (wheelEvent->deltaMode) {
+        case DOM_DELTA_PIXEL:
+          deltaY /= 100; // 100 pixels make up a step
```

Comment:
Sorry, missed this earlier but I believe upstream prefers `/*c-style comments*/`. Also indentation looks a little off.

Reply:
whoops, thanks